modules/chapter1/pages/install.adoc

```diff
@@ -0,0 +1,263 @@
++
+image::search-devhub-chart.png[title=Search for RHDH Helm Chart]
+
+. In the `Red Hat Developer Hub` chart page, click `Create`. You will see the `Create Helm Release` page with some fields that need to be edited.
+
+.. Change the `Release Name` to `rhdh`. Ensure that the latest helm chart is selected in the `Chart version` drop-down (As of Dec 2023, latest RHDH version is 0.2).
+
+.. Expand `Root Schema > global` and change the first field value from `apps.example.com` to the wildcard domain name of your OpenShift cluster. In RHDP, it is in the format `apps.cluster-<guid>.dynamic.opentlc.com`, where `<guid>` is unique to your provisioned OpenShift cluster.
+
+.. The final screen should look like the following:
++
+image::edit-helm-chart-values.png[title=Edit Helm Chart Details]
+
+.. Click `Create` to deploy the helm chart.
+
+. Wait for 10-15 minutes while the helm chart is deployed. The `PostgreSQL` and `Red Hat Developer Hub` should be fully started.
+Click the `Open URL` link to access Red Hat Developer Hub.
++
+image::rhdh-installed.png[RHDH Topology View]
+
+. You should now see the Red Hat Developer Hub log in page.
++
+image::rhdh-homepage.png[title=RHDH Log in Page]
+
+. Click `ENTER` in the `Guest` card to log in as a guest user. You will configure RHDH to authenticate against GitHub and other identity providers in later chapters of this course.
++
+image::rhdh-guest-user.png[title=Log in as a guest user]
++
+You have now successfully installed Red Hat Developer Hub on your OpenShift cluster.
+
+== Installation using Helm CLI
+
+IMPORTANT: You must complete the steps listed in the *Before Installation* section before using the steps below.
+
+You will need to install the `helm` CLI. You can download the *helm* and other CLI tools from the OpenShift web console by clicking the *"?"* icon on the top right navigation bar.
+
+image::cli-tools.png[title=Download CLI Tools]
+
+=== CLI Install Steps
+
+. Add the OpenShift helm chart repository
++
+```bash
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+```
+
+. Download the default *values.yaml* file for the RHDH helm chart
++
+```bash
+$ helm show values openshift-helm-charts/redhat-developer-hub > values.yaml
+```
+
+. Change the value of the *global.clusterRouterBase* attribute to the wildcard OpenShift DNS name, and also edit the *upstream.nameOverride* to *rhdh*.
++
+[subs=+quotes]
+----
+global:
+  clusterRouterBase: *apps.cluster-<guid>.dynamic.opentlc.com*
+  host: ""
+
+upstream:
+  nameOverride: *rhdh*
+  backstage:
+...
+----
+
+. Log in to the OpenShift cluster as the cluster administrator. Ensure that you switch to the *devhub* project.
++
+```bash
+$ oc login -u admin https://api.cluster-<your_guid>.dynamic.opentlc.com:6443
+$ oc project devhub
+```
+
+. Install the RHDH helm chart with the updated *values.yaml* file
++
+[subs=+quotes]
+----
+$ helm upgrade -i rhdh -f default-values.yaml openshift-helm-charts/redhat-developer-hub
```

Comment:
This step failed for me because the above step on line 147 specifies the file named `values.yaml` yet this command expected `default-values.yaml`

Reply:
good catch! fixed